lib/facility.rb

```diff
@@ -26,13 +26,13 @@ def register_vehicle(vehicle)
       vehicle.set_plate_type
       @collected_fees += vehicle.set_fees
     else
-      "service not provided at this location"
+      false
```

Comment:
I had originally written a sentence in response to else statement of the facility not having services, but then changed it to false to keep the same conventions as my other if/else statements